x/concentrated-liquidity/model/pool_test.go

```diff
@@ -41,6 +42,100 @@ func TestConcentratedPoolTestSuite(t *testing.T) {
 	suite.Run(t, new(ConcentratedPoolTestSuite))
 }
 
+// TestGetAddress tests the GetAddress method of pool
+func (s *ConcentratedPoolTestSuite) TestGetAddressAndGetIncentiveAddress() {
```

Comment:
This is just testing getAddress, not getIncentivesAddress

Reply:
Fixed test name correctly and changed to https://github.com/osmosis-labs/osmosis/pull/5266/commits/51fce1f3393cc2e43e9de0e707d86ad5ca561b34